crawler/twitter.js

```diff
@@ -32,6 +35,9 @@ twitter.getTweetsList = function(ids, retries) {
         ids = ids.toString();
     }
 
+
+exports.fetchTweets = function(id, query, maxID) {
```

Comment:
Your branch is still corrupt. Please start a clean branch from the current `develop` like I already said.